httprequest_lego_provider/tests/test_views.py

```diff
@@ -1,6 +1,9 @@
 # Copyright 2024 Canonical Ltd.
 # See LICENSE file for licensing details.
 """Unit tests for the views module."""
+
+# pylint:disable=imported-auth-user
```

Comment:
Is this new? For these and others, do you think it is worth including an explanatory comment why they are needed?

Reply:
Added